modules/packages/LinearAlgebra.chpl

```diff
@@ -1950,15 +1960,15 @@ proc svd(A: [?Adom] ?t) throws
   // Results
 
   // Stores singular values, sorted
-  var s: [1..min((...A.shape))] realType;
+  var s: [0..<min((...A.shape))] realType;
   // Unitary matrix, U
-  var u: [1..m, 1..m] t;
+  var u: [0..<m, 0..<m] t;
   // Unitary matrix V^T (or V^H)
-  var vt: [1..n, 1..n] t;
+  var vt: [0..<n, 0..<n] t;
 
   // if return code 'info' > 0, then this stores unconverged superdiagonal
   // elements of upper bidiagonal matrix 'B' whose diagonal is in 's'.
-  var superb: [1..min((...A.shape))-1] realType;
+  var superb: [0..#min((...A.shape))] realType;
```

Comment:
Shouldn't we keep `-1` here?

Reply:
Yes it seems so. I'll look into why tests didn't catch this and try to update if they should.

Reply:
Updated in next commit. It looks like this out argument is not used in the current implementation. I suppose it could be helpful debugging information if svd did not converge, but not even scipy svd bothers to do anything with it as a reference.